CHANGELOG.md

```diff
@@ -36,7 +36,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#3991](https://github.com/open-telemetry/opentelemetry-python/pull/3991))
 
 ## Version 1.25.0/0.46b0 (2024-05-30)
-
+- OTLP exporter is encoding invalid span/trace IDs in the logs fix
```

Comment:
Please keep the empty line and move it to unreleased section

Reply:
Move it under the fixed section of the unreleased section please

Reply:
I don't really see a fixed section, just "unreleased" and then "version 1.25.0/0.46b0".

Reply:
Sorry, confused with `-contrib`